src/greptimeai/tracker/openai_tracker.py

```diff
@@ -26,6 +26,7 @@ def setup(
     database: str = "",
     token: str = "",
     client: Optional[OpenAI] = None,
+    async_client: Optional[AsyncOpenAI] = None,
```

Comment:
since OpenAI Python lib assures that all methods are the same, what about change signature to:

`client: Optional[Openai, AsyncOpenAI] = None,`


ref: https://github.com/openai/openai-python#async-usage